modules/jupyter-widget/src/utils.js

```diff
@@ -23,9 +23,9 @@ export function updateDeck(inputJSON, {jsonConverter, deckConfig}) {
 }
 
 export function initDeck({mapboxApiKey, container, jsonInput}, onComplete, handleClick) {
-  require(['mapbox-gl', 'h3', 'S2'], mapboxgl => {
-    require(['deck.gl'], deckgl => {
-      try {
+  try {
```

Comment:
- If this is not related to the tooltip feature, make the change in another PR.
- try... catch does not apply to errors inside the callback.
- Make a comment on what might break.